packages/devtools_app/lib/debugger_html_plugin.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// We actually use dart:html here, no shim.
+// This code may only be compiled into the web app.
+import 'dart:ui' as web_ui;
+
+import 'package:devtools_app/src/ui/html_elements.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'package:html_shim/html.dart' as html;
+
+import 'src/debugger/html_debugger_screen.dart';
+import 'src/framework/html_framework.dart';
```

Comment:
including all of html_framework.dart worries me a bit as there is going to be a bunch of intentional overlap with the flutter framework.

Reply:
We need to quantify exactly which functionality needs to get tied in with Flutter.  We can discuss this in detail in https://github.com/flutter/devtools/issues/1259